crates/primitives/src/net.rs

```diff
@@ -43,24 +43,13 @@ pub static HOLESKY_BOOTNODES : [&str; 2] = [
 ];
 
 #[cfg(feature = "optimism")]
-/// OP Mainnet Bootnodes
-pub static OP_BOOTNODES: [&str; 3] = [
+/// OP stack mainnet boot nodes.
+pub static OP_BOOTNODES: &[&str] = &[
+    // OP Labs
     "enode://ca2774c3c401325850b2477fd7d0f27911efbf79b1e8b335066516e2bd8c4c9e0ba9696a94b1cb030a88eac582305ff55e905e64fb77fe0edcd70a4e5296d3ec@34.65.175.185:30305",
     "enode://dd751a9ef8912be1bfa7a5e34e2c3785cc5253110bd929f385e07ba7ac19929fb0e0c5d93f77827291f4da02b2232240fbc47ea7ce04c46e333e452f8656b667@34.65.107.0:30305",
     "enode://c5d289b56a77b6a2342ca29956dfd07aadf45364dde8ab20d1dc4efd4d1bc6b4655d902501daea308f4d8950737a4e93a4dfedd17b49cd5760ffd127837ca965@34.65.202.239:30305",
-];
-
-#[cfg(feature = "optimism")]
-/// OP Testnet Bootnodes
-pub static OP_TESTNET_BOOTNODES: [&str; 3] = [
```

Comment:
this should be proposed on this PR, not here 

https://github.com/paradigmxyz/reth/pull/7990 

i suggested another approach, since i like the separation

Reply:
the separation is only based on which org hosts though, it's not a programmatic one. how come you want the separation?

Reply:
oh yeah this got dragged in